Module not found errors after upgrading to 1.1.6

We previously were using ^1.1.5. After last evening, our builds started failing with seemingly weird errors

```
resolve failed:  { Error: Cannot find module 'mz'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.requireRelative.resolve (f:\git\myapp\node_modules\require-relative\index.js:30:17)
    at resolve (f:\git\myapp\node_modules\modify-babel-preset\lib\serialize.js:26:26)
    at findAndRemove (f:\git\myapp\node_modules\modify-babel-preset\lib\serialize.js:83:11)
    at f:\git\myapp\node_modules\modify-babel-preset\lib\serialize.js:126:13
    at Array.map (native)
    at loadPreset (f:\git\myapp\node_modules\modify-babel-preset\lib\serialize.js:118:29)
    at module.exports (f:\git\myapp\node_modules\modify-babel-preset\index.js:95:19)
    at Object.<anonymous> (f:\git\myapp\node_modules\babel-preset-save\index.js:3:35)
    at Module._compile (module.js:570:32)
    at Module.replacementCompile (f:\git\myapp\node_modules\nyc\node_modules\append-transform\index.js:63:13)
    at module.exports (f:\git\myapp\node_modules\nyc\node_modules\default-require-extensions\js.js:8:9)
    at f:\git\myapp\node_modules\nyc\node_modules\append-transform\index.js:67:4
    at require.extensions.(anonymous function) (f:\git\myapp\node_modules\babel-register\lib\node.js:152:7)
    at Object.<anonymous> (f:\git\myapp\node_modules\nyc\node_modules\append-transform\index.js:67:4)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at f:\git\myapp\node_modules\babel-core\lib\transformation\file\options\option-manager.js:302:17
    at Array.map (native)
    at OptionManager.resolvePresets (f:\git\myapp\node_modules\babel-core\lib\transformation\file\options\option-manager.js:270:20)
    at OptionManager.mergePresets (f:\git\myapp\node_modules\babel-core\lib\transformation\file\options\option-manager.js:259:10)
    at OptionManager.mergeOptions (f:\git\myapp\node_modules\babel-core\lib\transformation\file\options\option-manager.js:244:14)
    at OptionManager.init (f:\git\myapp\node_modules\babel-core\lib\transformation\file\options\option-manager.js:374:12)
    at compile (f:\git\myapp\node_modules\babel-register\lib\node.js:103:45)
    at loader (f:\git\myapp\node_modules\babel-register\lib\node.js:144:14)
    at require.extensions.(anonymous function) (f:\git\myapp\node_modules\babel-register\lib\node.js:154:7)
    at Object.<anonymous> (f:\git\myapp\node_modules\nyc\node_modules\append-transform\index.js:67:4)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (f:\git\myapp\test\helpers\spec-helper.js:9:1)
    at Module._compile (module.js:570:32)
    at Module.replacementCompile (f:\git\myapp\node_modules\nyc\node_modules\append-transform\index.js:63:13)
    at module.exports (f:\git\myapp\node_modules\nyc\node_modules\default-require-extensions\js.js:8:9)
    at Object.<anonymous> (f:\git\myapp\node_modules\nyc\node_modules\append-transform\index.js:67:4)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at f:\git\myapp\node_modules\mocha\bin\_mocha:311:3
    at Array.forEach (native)
    at Object.<anonymous> (f:\git\myapp\node_modules\mocha\bin\_mocha:310:10)
    at Module._compile (module.js:570:32)
    at Module.replacementCompile (f:\git\myapp\node_modules\nyc\node_modules\append-transform\index.js:63:13)
    at module.exports (f:\git\myapp\node_modules\nyc\node_modules\default-require-extensions\js.js:8:9)
    at Object.<anonymous> (f:\git\myapp\node_modules\nyc\node_modules\append-transform\index.js:67:4)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Function.Module.runMain (module.js:604:10)
    at Function.runMain (C:\Users\mrchief\.node-spawn-wrap-12100-9c1de388738c\node:40:10)
    at Object.<anonymous> (f:\git\myapp\node_modules\nyc\bin\wrap.js:20:4)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Function.Module.runMain (module.js:604:10)
    at Object.<anonymous> (C:\Users\mrchief\.node-spawn-wrap-12100-9c1de388738c\node:140:8)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.runMain (module.js:604:10)
    at run (bootstrap_node.js:394:7)
    at startup (bootstrap_node.js:149:9)
    at bootstrap_node.js:509:3 code: 'MODULE_NOT_FOUND' } mz
```

Similar errors were thrown for `font-awesome`, `caniuse-db`, `babel-runtime`, `webpack-core`, `alt-utils`. We don't even have `mz` or `caniuse-db` as a dependency.

Ditching this preset and switching to this fixes the issue so I'm thinking its probably one of the other plugins/presets causing this.

**.babelrc**
```
  "presets": ["react", "es2015", "stage-0"],
  "plugins": [
    "add-module-exports",
    "transform-decorators-legacy",
    ["transform-class-properties", { "spec": true }]
  ],
```

--- a/index.js
+++ b/index.js
@@ -1,16 +1,13 @@
 module.exports = {
   presets: [
-    require('modify-babel-preset')('babel-preset-es2015', {
-      'transform-es2015-classes': { loose: true },
-      'transform-es2015-modules-commonjs': { loose: true }
-    }),
+    require('babel-preset-es2015'),
     require('babel-preset-react'),
     require('babel-preset-stage-0')
   ],
   plugins: [
     require('babel-plugin-transform-decorators-legacy').default,
     require('babel-plugin-add-module-exports'),
-    [require('babel-plugin-transform-es2015-template-literals'), { spec: true }],
+    [ require('babel-plugin-transform-es2015-template-literals'), { spec: true } ],
     require('babel-plugin-transform-es3-member-expression-literals'),
     require('babel-plugin-transform-es3-property-literals'),
     require('babel-plugin-transform-jscript'),

--- a/package.json
+++ b/package.json
@@ -18,7 +18,6 @@
     "babel-plugin-transform-proto-to-assign": "^6.5.0",
     "babel-preset-es2015": "^6.5.0",
     "babel-preset-react": "^6.5.0",
-    "babel-preset-stage-0": "^6.5.0",
-    "modify-babel-preset": "^3.2.0"
+    "babel-preset-stage-0": "^6.5.0"
   }
 }

